### PR TITLE
Add release notes for 2021.1.0 Eos.

### PR DESCRIPTION
**Pull request by digisomni**
Pending changelog for Iamus + potential additions to it and the Dashboard in collaboration depending on timelines.

### CODE REVIEW

File: docs/source/release-notes.rst [hunk 1]
```diff
@@ -8,6 +8,7 @@ See what's been released in Vircadia.
     :maxdepth: 2
     :titlesonly:
 
+    Release 2021.1.0 Eos (2021-02-05) <release-notes/2021-1-0-Eos>
```
Comment (ctrlaltdavid) [inline docs/source/release-notes.rst:11]: Update date at time of release. : )

File: docs/source/release-notes/2021-1-0-Eos.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# Alpha 2021.1.0 Eos Release Notes
+
+This version of Vircadia is a new release. It is **not** compatible with the 2020.3.0 Demeter protocol. This version was released on February 5th, 2021.
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2021-1-0-Eos.md:3]:
```suggestion
This version of Vircadia is a new release. It was released on February 5th, 2021.

It has a protocol change. It is **not** compatible with the 2020.3.n Demeter protocol. 
```
Also, update date at time of release.

File: docs/source/release-notes/2021-1-0-Eos.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+* App: Allow any user to teleport to any user within a domain using the 'People' app. (#991)
+* API: Add `releaseName` property to the `About` API. (#996)
+
+#### Miscellaneous
+
+* Application: Add functionality to skip repeated log entries. (#937)
+* Docs: List required dependency for Visual Studio on Windows builds. (#967)
+* Application: Fix multiple crashes when using VR with a QML disabled build. (#971)
+* Application: Vircadia now accepts the `RELEASE_NAME` env variable. (#975)
+* Build: Updated Windows VCPKG to use an alternative JOM download URL instead of Qt's. (#983)
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2021-1-0-Eos.md:36]:
```suggestion
* Build: Updated Windows VCPKG to include an alternative JOM download URL instead of just Qt's. (#983)
```

File: docs/source/release-notes/2021-1-0-Eos.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+## Vircadia Launcher (Codename Pantheon)
+
+[Vircadia's launcher](https://github.com/vircadia/pantheon-launcher) updates can be found [here](https://github.com/vircadia/pantheon-launcher/commits/master). The launcher is on version 1.0.1.
+
+* Enable tray functionality on launcher.
+* Set 'News' tab as default.
+* Add 'Metaverse' tab.
+* Add 'Hide On Launch' option to minimize the launcher to tray when Interface is started successfully.
+* Various backend & stability fixes have been implemented.
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2021-1-0-Eos.md:74]:
```suggestion
* Various backend & stability fixes.
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

File: docs/source/release-notes.rst [hunk 5]
```diff
@@ -8,6 +8,7 @@ See what's been released in Vircadia.
     :maxdepth: 2
     :titlesonly:
 
+    Release 2021.1.0 Eos (2021-02-12) <release-notes/2021-1-0-Eos>
```
Comment (JulianGro) [inline docs/source/release-notes.rst:11]: I can understand moving away from "Alpha", but I feel like we are in "Beta" rather than a full release.
```suggestion
    Beta Release 2021.1.0 Eos (2021-02-12) <release-notes/2021-1-0-Eos>
```

File: docs/source/release-notes/2021-1-0-Eos.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+This version of Vircadia is a new release. It is **not** compatible with the 2020.3.0 Demeter protocol. This version was released on February 12th, 2021.
+
+## Interface (Codename Athena)
+
+The numbers at the end of each item are the PR numbers in the Vircadia [repo](https://github.com/vircadia/vircadia).
+
+#### General
+
+* UI/UX: The VR controls for regular and fullbody users has been overhauled. (#928)
+    1. Fix being unable to sit in the real world in VR without the avatar and viewpoint being popped up to a standing position (fixes https://github.com/vircadia/vircadia/issues/825). With the new option "Allow my avatar to stand", we can choose to have the avatar stand only when the user is standing. The default behavior is the same as before (always allowed to stand; allowed to lean only when user is standing), except that now, when foot trackers are used, the avatar is never made to stand artificially.
```
Comment (JulianGro) [inline docs/source/release-notes/2021-1-0-Eos.md:12]: Most of our PRs fix issues so starting to include them seems like a bad idea, unless we do a similar thing like on the PR numbers.
```suggestion
    1. Fix being unable to sit in the real world in VR without the avatar and viewpoint being popped up to a standing position. With the new option "Allow my avatar to stand", we can choose to have the avatar stand only when the user is standing. The default behavior is the same as before (always allowed to stand; allowed to lean only when user is standing), except that now, when foot trackers are used, the avatar is never made to stand artificially.
```

File: docs/source/release-notes/2021-1-0-Eos.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+    2. Fix very wrong walk/fly directions when using foot tracking or when using the mode that allows the avatar to sit when the user sits (previously "Avatar lean behavior: disabled").
+    3. Fix VR walk speeds being too slow at small avatar scale and too fast at large avatar scale (fixes https://github.com/vircadia/vircadia/issues/851). The avatar scale was being factored-in twice in the velocity calculation.
```
Comment (JulianGro) [inline docs/source/release-notes/2021-1-0-Eos.md:14]: Same as above.
```suggestion
    3. Fix VR walk speeds being too slow at small avatar scale and too fast at large avatar scale. The avatar scale was being factored-in twice in the velocity calculation.
```

File: docs/source/release-notes/2021-1-0-Eos.md [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+    8. Fix for incorrect vertical position of the avatar and viewpoint after changing recenter mode while not standing in the real world.
+    9. The logic controlling the direction of `MyAvatar`'s action motor was hard to follow and maintain, as it was split between two functions. Now it's clearer and all done in `calculateScaledDirection`.
+    10. Leaning and lean recentering now work as intended at any avatar scale.
+* API: Add `useOriginalPivot` property to model entities allowing entities to use their original pivot. (#951)
+* Graphics: Improve rendering of transparent entities. (#953)
+* API: Add `billboardMode` property for all entity types allowing entities to face the user as specified in the following modes: none, yaw, full. (#961)
+* App: The 'Appreciate' app has been added as a default app to Interface. (#974)
+* API: Add `userAgent` property to web entities allowing the web entity to use the specified user agent when browsing. (#998, #1017)
+* App: Allow any user to teleport to any user within a domain using the 'People' app. (#991)
```
Comment (JulianGro) [inline docs/source/release-notes/2021-1-0-Eos.md:27]: Is this right? Teleport permissions have been removed?

File: docs/source/release-notes/2021-1-0-Eos.md [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+
+#### Miscellaneous
+
+* Application: Add functionality to skip repeated log entries. (#937)
+* Docs: List required dependency for Visual Studio on Windows builds. (#967)
+* Application: Fix multiple crashes when using VR with a QML disabled build. (#971)
+* Application: Vircadia now accepts the `RELEASE_NAME` env variable. (#975)
+* Build: Updated Windows VCPKG to include an alternative JOM download URL instead of just Qt's. (#983)
+* Build: Revise Windows build docs to improve formatting and list Node.JS as a dependency. (#987)
+* Misc: 'Alpha' references have been removed for Interface indicating that Vircadia is now a full release. (#996)
```
Comment (JulianGro) [inline docs/source/release-notes/2021-1-0-Eos.md:38]: We are skipping beta? Even though there is tons of bugs and reliability issues?

**Review by JulianGro — CHANGES_REQUESTED** [review]

File: docs/source/release-notes/2021-1-0-Eos.md (hunk 8 above)
Comment (ctrlaltdavid) [reply docs/source/release-notes/2021-1-0-Eos.md:27]: Unless I am mistaken, there aren't any teleport permissions per se. Rather there was a limitation in that you had to be logged in order for the "teleport to user" function to work.

File: docs/source/release-notes.rst (hunk 5 above)
Comment (ctrlaltdavid) [reply docs/source/release-notes.rst:11]: At a developer or community meeting not so long ago, a general consensus was reached to to remove such labels - the software is what it is. And it has been released. It's certainly usable; fine for some use cases, room for improvement in other use cases.

File: docs/source/release-notes/2021-1-0-Eos.md (hunk 6 above)
Comment (ctrlaltdavid) [reply docs/source/release-notes/2021-1-0-Eos.md:12]: Good point. The issue numbers should be in the PRs, anyway.

File: docs/source/release-notes/2021-1-0-Eos.md [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+* API: Add `userAgent` property to web entities allowing the web entity to use the specified user agent when browsing. (#998, #1017)
+* App: Allow any user to teleport to any user within a domain using the 'People' app. (#991)
+* API: Add `releaseName` property to the `About` API. (#996)
+
+#### Miscellaneous
+
+* Application: Add functionality to skip repeated log entries. (#937)
+* Docs: List required dependency for Visual Studio on Windows builds. (#967)
+* Application: Fix multiple crashes when using VR with a QML disabled build. (#971)
+* Application: Vircadia now accepts the `RELEASE_NAME` env variable. (#975)
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2021-1-0-Eos.md:35]:
```suggestion
* Build: Vircadia now accepts the `RELEASE_NAME` env variable. (#975)
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

File: docs/source/release-notes.rst (hunk 5 above)
Comment (digisomni) [reply docs/source/release-notes.rst:11]: Yeppers, basically the idea is that we have to be **very** clear on what parts of Vircadia are capable/ready for what use cases.

This page is meant to help indicate this https://vircadia.com/what-is-vircadia/ by outlining what it is and what it isn't respectively. I'll keep at updating it regularly especially given feedback on the various parts of the system.

File: docs/source/release-notes/2021-1-0-Eos.md (hunk 8 above)
Comment (JulianGro) [reply docs/source/release-notes/2021-1-0-Eos.md:27]: No. It is supposed to abide to the permissions set by the user. The setting is still there in v2021.1.0-rc2:
1Auswahl_846

File: docs/source/developer/build/BUILD_ANDROID.md [hunk 11]
```diff
@@ -1,8 +1,8 @@
 # Build Android
 
-*Last Updated on December 21, 2019*
+*Last Updated on December 15, 2020*
```
Comment (ctrlaltdavid) [inline docs/source/developer/build/BUILD_ANDROID.md:3]: Is this date correct?

File: docs/source/release-notes/2021-1-0-Eos.md [hunk 12]
```diff
@@ -19,13 +19,19 @@ The numbers at the end of each item are the PR numbers in the Vircadia [repo](ht
...
-* API: Add `billboardMode` property for all entity types allowing entities to face the user as specified in the following modes: none, yaw, full. (#961)
+* API: Add `billboardMode` property for all entity types allowing entities to face the user as specified in the following modes: none, yaw, full. (#961, #1041)
 * App: The 'Appreciate' app has been added as a default app to Interface. (#974)
 * API: Add `userAgent` property to web entities allowing the web entity to use the specified user agent when browsing. (#998, #1017)
 * App: Allow any user to teleport to any user within a domain using the 'People' app. (#991)
 * API: Add `releaseName` property to the `About` API. (#996)
+* API: Add `alignment` property to text entities allowing text to be aligned to 'left', 'center', and 'right'. (#1020)
+* App: 'Create' app February 2021 improvements package. (#1030)
+    * The Entity List menubar has been reorganized.
+        - Introducing a new menu “**Transform**”. The menu items have been re-dispatched. CSS has been retouched to display clearly which menu is open.
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2021-1-0-Eos.md:32]:
```suggestion
        - Introducing a new menu “Transform”. The menu items have been re-dispatched. CSS has been retouched to display clearly which menu is open.
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

File: docs/source/developer/build/BUILD_ANDROID.md (hunk 11 above)
Comment (digisomni) [reply docs/source/developer/build/BUILD_ANDROID.md:3]: Yeppers, it's all imported from the main repo so if it's incorrect then it should be fixed there.
